build.sbt

```diff
@@ -52,6 +52,7 @@ lazy val baseSettings: Project => Project =
       scalaJSLinkerConfig := scalaJSLinkerConfig.value.withSourceMap(false),
       /* for slinky */
       libraryDependencies ++= Seq("me.shadaj" %%% "slinky-hot" % "0.6.5"),
+      libraryDependencies += "me.shadaj" %%% "slinky-web" % "0.6.5",
```

Comment:
this shouldn't be necessary, or?

Reply:
Indeed, I got this working after several tries